pints/_transformation.py

```diff
@@ -264,24 +264,22 @@ def to_search(self, p):
         """
         raise NotImplementedError
 
+    def elementwise(self):
+        r"""
+        Returns True if the transformation is element-wise.
 
-class ElementWiseTransformation(Transformation):
-    r"""
-    Abstract base class for objects that provide *element-wise* transformations
-    between two parameter spaces: the model parameter space and a search space.
-    That is, for a parameter vector :math:`\boldsymbol{p}` in the model space
-    and a parameter vector :math:`\boldsymbol{q}` in the search space, the
-    transformation :math:`\boldsymbol{f}` relating the two vectors takes the
-    form
-
-    .. math::
-        q_i = f_i(p_i),
+        Element-wise transformation is a transformation :math:`\boldsymbol{f}`
```

Comment:
how about "An element-wise transformation is a transformation f that can be carried out element by element: EQUATION (as opposed to a transformation in which multiple elements are combined to create the transformed elements"

Reply:
Sounds good, updated!